packages/ui/src/components/button/button.native.tsx

```diff
@@ -1,8 +1,9 @@
 import { TouchableOpacity, ViewStyle } from 'react-native';
 
 import { ResponsiveValue } from '@shopify/restyle';
-import { Theme } from 'native';
 
+// FIXME issues#64: This is a temporary fix to commit as I broke the direct import of the theme
+import { Theme } from '../../theme-native';
 import { Box } from '../box/box.native';
 import { Text } from '../text/text.native';
```

Comment:
When did we decide on `.native` as the convention? Is that a RN standard I've forgotten?

Reply:
We want to co-locate component code so Edgar had split the files into `.web` for web and `.native` for RN. 

The plan is to co-locate so we can share as much as we can but I'm not sure how much that will be

Reply:
I know abt the colocation, was asking abt the naming convention of the files ...wasn't sure why `.native` was being used vs `.mobile`?